include/eraftpb.proto

```diff
@@ -80,6 +80,12 @@ message Message {
     uint64 reject_hint = 11;
     bytes context = 12;
     uint64 priority = 14;
+    // If this new field is not set, then use the above old field; otherwise
+    // use the new field. When broadcasting request vote, both fields are
+    // set if the priority is larger than 0. This change is not a fully
+    // compatible change, but it makes minimal impact that only new priority
+    // is not recognized by the old nodes during rolling update.
+    int64 priority_v2 = 15;
```

Comment:
I suggest to change this to `priority` and the old one to `deprecated_priority`.